src/views/Intervention/Intervention.jsx

```diff
@@ -0,0 +1,252 @@
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { useNavigate, useParams, useLocation } from 'react-router';
+import { useTracking } from 'react-tracking';
+
+import { Pager, NoResults, ResultsList, Spinner } from '../../components';
+import { useAppPaths, useCustomQuery } from '../../hooks';
+import { getClinicalTrials } from '../../services/api/actions';
+import { useStateValue } from '../../store/store';
+
+import {
+	appendOrUpdateToQueryString,
+	getKeyValueFromQueryString,
+	getPageOffset,
+	TokenParser,
+} from '../../utils';
+
+const Intervention = ({ data: [data] }) => {
+	const { codeOrPurl } = useParams();
```

Comment:
I did not notice this in Disease, but If we have data, do we really need codeOrPurl? Since data will either have the prettyUrl or concept ids. It would be one less hook called.